### PR TITLE
Update debugging_rails_applications.md [ci_skip]

### PR DESCRIPTION
**Pull request by thiagoaugusto**

### CODE REVIEW

File: guides/source/debugging_rails_applications.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ It can also be useful to save information to log files at runtime. Rails maintai
 
 Rails makes use of the `ActiveSupport::Logger` class to write log information. Other loggers, such as `Log4r`, may also be substituted.
 
-You can specify an alternative logger in `environment.rb` or any other environment file, for example:
+You can specify an alternative logger in `config/environment.rb` or any other environment file, for example:
```
Comment (rafaelfranca) [inline guides/source/debugging_rails_applications.md]: This should be `config/application.rb`

File: guides/source/debugging_rails_applications.md [hunk 2]
```diff
@@ -109,7 +109,7 @@ It can also be useful to save information to log files at runtime. Rails maintai
...
 
 ```ruby
 Rails.logger = Logger.new(STDOUT)
```
Comment (rafaelfranca) [inline guides/source/debugging_rails_applications.md]: Here should be `config.logger`

Comment (rafaelfranca) [inline guides/source/debugging_rails_applications.md]: And below should be `Rails.application.config.logger` but I think that may not work so better to check.

**Comment by rafaelfranca** [comment]
@thiagoaugusto do you want to keep working in this PR. Otherwise I'll have to close it.

**Comment by prathamesh-sonpatki** [comment]
@thiagoaugusto Are you working on the feedback? If not please let us know so somebody else can work on it.

**Comment by prajakta-tambe** [comment]
I am working on the suggested changes.

**Comment by prathamesh-sonpatki** [comment]
Closed by https://github.com/rails/rails/pull/23902

**Pull request closed without merging**